app/api.py

```diff
@@ -5,6 +5,7 @@
 from fastapi.responses import JSONResponse
 
 from app.data import MongoDB
+from app.utilities import *
```

Comment:
While using the wildcard is ok, to make the imports more readable consider changing the wildcard to the actual utilities you want to import.